modules/simple_forms_api/app/services/simple_forms_api/notification_email.rb

```diff
@@ -47,6 +49,11 @@ def send
 
     private
 
+    def check_missing_keys(config)
+      missing_keys = %i[form_data form_number confirmation_number].select { |key| config[key].nil? }
```

Comment:
So sorry, but I think we'd probably want this instead:

```suggestion
      missing_keys = %i[form_data form_number confirmation_number].select { |key| config[key] && config[key].nil? }
```